src/installationmanager.cpp

```diff
@@ -369,7 +369,7 @@ QString InstallationManager::generateBackupName(const QString &directoryName) co
 }
 
 
-bool InstallationManager::testOverwrite(GuessedValue<QString> &modName, bool *merge) const
+bool InstallationManager::testOverwrite(GuessedValue<QString> &modName, bool *merge)
```

Comment:
Removing the `const` is only needed to emit `modReplaced`, right? Isn't it possible to mark the signal as `const` to avoid this?

Reply:
Yeah but I removed it since it's not really truthful. It's deleting a mod folder, rewriting the ini and it's changing the download state. I know this doesn't strictly violate what const means but the signal might do all kinds of things in the future. 
When I first glanced at the function I assumed it only asked the user, I wouldn't have thought it also deleted the folder and all that.

Reply:
Yeah, I get the idea. Maybe renaming `testOverwrite` to something more meaningful would be better?

Reply:
I'm short on ideas for that one